media/js/base/global.js

```diff
@@ -34,15 +34,16 @@ function init_download_links() {
 }
 
 function update_download_text_for_old_fx() {
-    // if using an out of date firefox
-    if (isFirefox() && !isFirefoxUpToDate()) {
-        // look at each button to see if it's set to check for old firefox
-        $('.download-button').each(function() {
-            var $button = $(this);
-
-            if ($button.hasClass('download-button-check-old-fx')) {
+    // look at each button to see if it's set to check for old firefox
+    var $buttons = $('.download-button-check-old-fx');
```

Comment:
We could also move this selector inside the `isFirefox` conditional to avoid querying the DOM for other browsers?